sdk/resourcemanager/azure-resourcemanager-appservice/pom.xml

```diff
@@ -48,8 +48,6 @@
       --add-opens com.azure.resourcemanager.resources/com.azure.resourcemanager.resources=ALL-UNNAMED
       --add-opens com.azure.resourcemanager.resources/com.azure.resourcemanager.resources.fluentcore.arm=ALL-UNNAMED
       --add-opens com.azure.resourcemanager.storage/com.azure.resourcemanager.storage=ALL-UNNAMED
-
-      --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
```

Comment:
@lmolkova 

Reason why this is deleted?

We will get
```
WARN io.netty.util.concurrent.AbstractEventExecutor - A task raised an exception. Task: reactor.netty.resources.DefaultPooledConnectionProvider$DisposableAcquire@3d46e713
java.lang.IllegalAccessError: class com.azure.core.http.netty.NettyAsyncHttpClient (in unnamed module @0x32eff876) cannot access class com.azure.core.implementation.util.BinaryDataHelper (in module com.azure.core) because module com.azure.core does not export com.azure.core.implementation.util to unnamed module @0x32eff876
```
if this is removed.